templates_qc/example_fov_grid_generate.ipynb

```diff
@@ -1,5 +1,12 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Lets rework the notebook to assume it is specifically for annotating existing TMAs, not for generating tiled images. All of the functions and arguments for options between the two should be removed. 

Organization wise, all of the filepaths should be defined at once in the top notebook. Similar to pixel clustering, let's have the default be a single identifier, like "example_TMA", and then everything named based on that: identifier + '_manual_fovs.json', identifier + '_automatic_fovs.json', identifier + '_mapping.json', etc.

Reply:
Once I'm done addressing the other comments above, I'll work on separating this into two separate notebooks.